src/cppcache/src/CacheXmlParser.cpp

```diff
@@ -1165,7 +1166,7 @@ void CacheXmlParser::startPersistenceManager(const xmlChar** atts) {
         throw CacheXmlException(s.c_str());
       }
 
-      ACE_OS::strncpy(libraryFunctionName, (char*)atts[i], len);
+      std::strncpy(libraryFunctionName, (char*)atts[i], len);
```

Comment:
I am of the camp that believes that large efforts like that are rarely undertaken unless there is an easy tool to do the work for you. The clang-tidy rule fixed some when we did the initial reformat and cleanup to Google C++ Style Guide. There were some it was unable to do automagically, my guess because they were ambiguous. I would make an effort that any file you edit should compile clean, no warnings and no static analyzer warnings/errors. You can enable clang-tidy by passing CLANG_TIDY_ENABLED=ON to CMake.